File: sentimental-mario-less/mario.py
from cs50 import get_int
# Prompot user for the height
height = get_int("Height: ")
# Make sure the height is from 1 to 8
while True:
    if height in range(1, 9):
        break
    else:
        height = get_int("Height: ")
        
# Main loop for the height
for i in range(height):
    for j in range(height-1, i, -1):
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print()

File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    // Prompt user for height
    do
    {
        height = get_int("Enter Height: ");

    }
    while (!(height > 0 && height < 9));

    // Iteration for number of lines
    for (int i = 1 ; i <= height ; i++)
    {

        // first triangle
        // Iteration for number of spaces
        for (int s = height - i ; s > 0 ; s--)
        {
            printf(" ");
        }
        // Iteration for number of #
        for (int j = 1 ; j <= i ; j++)
        {
            printf("#");
        }

        printf("  ");
        // Second triangle
        // Iteration for number of #
        for (int j = 1 ; j <= i ; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}